src/pages-and-resources/discussions/app-config-form/apps/lti/LtiConfigForm.jsx

```diff
@@ -132,8 +117,23 @@ function LtiConfigForm({ onSubmit, intl, formRef }) {
               )}
             </Form.Group>
           </>
+        ) : (
+          <p>
+            <FormattedMessage
+              {...messages.stuffOnlyConfig}
+              values={{
+                providerName,
+                platformName: getConfig().SITE_NAME,
+                supportEmail: supportEmail ? (
+                  <MailtoLink to={supportEmail}>{supportEmail}</MailtoLink>
+                ) : (
+                  'support'
+                ),
+              }}
+            />
+          </p>
         )}
-        {piiConfig.piiSharing && (
+        {(enablePIISharing) && (
```

Comment:
enablePIISharing will always be false?

Reply:
Yes. PII sharing will not be visible to any user (global staff, educators). I have confirmed this requirement with aamir.